reco_utils/azureml/wide_deep.py

```diff
@@ -33,11 +33,17 @@
 
 
 def _log(metric, value):
```

Comment:
Shouldn't this method be public (with run as an argument) so that it is used in other python files in the azureml directory?

Reply:
This wrapper is necessary only when we use a notebook with papermill on AzureML and want to record the same metrics multiple times within the notebook-context, e.g. record training error for every epoch. Because papermill will overwrite the values whenever we record by using the same key, we had to store the values into a list and record the list with the key. Once the notebook run is done, AzureML script (which is in this case, `azureml/wide_deep.py`) gets the list from papermill and write to AzureML via this `_log` function.

This approach is temporal because AzureML is planning to support papermill / notebooks sometime soon. If the day come, we will replace this method :-) So, I would keep this method private unless we have other examples that need this approach before the AzureML's update.